### DISCUSSION TITLE
Github.dev vscode error previewing markdown

### DISCUSSION THREAD

**Discussion by bmuellerhstat** [community/community#37398] [Other Feature Feedback, Questions, & Ideas]
Several of my students are having an issue with previewing markdown using Github.dev. The error says:
```
Error loading webview: Error: Could not register service workers: NotSupportedError: Failed to register a ServiceWorker for scope ('https://...)
The user denied permission to use Service Worker...
```
Any ideas on how to fix this? Thanks in advance!

Brian

<img width="1058" alt="githubdev" src="https://user-images.githubusercontent.com/14807115/198142606-aa4e50d1-fa30-46b3-8342-defdf35c6df3.png">

**Comment by exkludera** [comment]
https://github.com/orgs/community/discussions/26316
https://stackoverflow.com/questions/72498891/github-codespaces-error-loading-webview-error-could-not-register-service-worke

**Reply by bmuellerhstat** [reply]
Thank you so much!
